app/helpers/compute_resources_vms_helper.rb

```diff
@@ -114,6 +114,20 @@ def available_actions(vm, authorizer = nil)
     end
   end
 
+  def common_available_actions(vm, authorizer = nil)
+    actions = []
+    actions << vm_delete_action(vm, authorizer)
+    actions << vm_console_action(vm)
+    host_action = vm_host_action(vm)
+    if host_action
+      actions << host_action
+    else
+      actions << vm_import_action(vm, :class => 'btn btn-default')
```

Comment:
is importing supported by all compute resource providers?

Reply:
Yep. Just the amount of automatically filled attributes differs.
We have tests for every CR in core: https://github.com/theforeman/foreman/blob/develop/test/unit/compute_resource_host_importer_test.rb